modules/combined/tests/generalized_plane_strain_tm_contact/generalized_plane_strain_tm_contact_field.i

```diff
@@ -2,7 +2,7 @@
   order = FIRST
```

Comment:
Does it work with 'quadrature = true'? If it does, we should switch to running it that way because that is preferable.